### ISSUE TITLE
determine compatibility; also, I did research

### ISSUE THREAD

**Issue by GiselleSerate** [GiselleSerate/myaliases#36]
While tagging this repo, I discovered some other repos in the [bashrc tag](https://github.com/topics/bashrc) and others that might be helpful for figuring out what's actually necessary to be compatible with. Examples: 

- https://github.com/voku/dotfiles 
(Cygwin, Git Bash)
- https://github.com/Bash-it/bash-it
(okay this is basically us, that's worrying lol)

Did we happen to check if these commands/aliases are already a thing?

**Comment by aryam7** [comment]
Yeah, I've actually been reading up on a lot of other similar projects, especially for cds.
There's some really great stuff out there. Example: https://github.com/junegunn/fzf
I think cds is unique in that we attempted to make it an extension of cd. I haven't seen any other attempts like that. I also like how simple the cds command is. It's nice when we keep things short and sweet.

**Comment by GiselleSerate** [comment]
I am definitely a fan of cds. I think it's awesome and haven't found anything like it either. I mean, the particular format and the concept of having all your dotfiles in a repo is definitely not new, but I don't think we should be particularly worried about that. We do seem to have something unique to offer here.

**Comment by GiselleSerate** [comment]
Useful thread for compatibility: https://stackoverflow.com/questions/11376975/is-there-a-minimally-posix-2-compliant-shell
Sounds like we should use dash or posh to check for compatibility, although there's no good solution besides checking a bunch of shells.

**Comment by GiselleSerate** [comment]
What we probably want to do: 
Go through common shells and test all our commands on them. As we test each command on a system, add the appropriate badge in the wiki. If it works for every command in a file, add it at the top, next to the filename in the wiki. If it only works for some, add the badge next to each command's name. (Or maybe above, or maybe under? Haven't decided, some of these commands are long.) 
In approximate order of priority: 
- [x] wsl
- [ ] ubuntu (use VM)
- [ ] zsh
- [ ] osx
